src/Microsoft.PowerShell.ScheduledJob/ScheduledJobDefinition.cs

```diff
@@ -1883,10 +1883,7 @@ public void Dispose()
 
         private static int GetCurrentId()
```

Comment:
We don't compile the scheduledjob code in GitHub repo, so please revert the changes here.